esp/src/src/nls/hpcc.ts

```diff
@@ -354,7 +354,7 @@ export = {
         GetLogicalFilePart: "Logical File Part",
         GetProtectedList: "Protected List",
         GetValue: "Value",
-        GetPart: "Part",
+        GetPart: "Get Part",
```

Comment:
Did you check for other places this might be used?

Reply:
I did. This change was reverting it back to the original string. It was changed in a PR last year (https://github.com/hpcc-systems/HPCC-Platform/commit/751d960e8aa5c6b241b997e671e79ec0e499efb9#diff-f144f03b8fefab4eb1ac1b145d7691b679268d4c9d7251f7c44f12895e47c27bL342). I missed it when I reviewed that at the time.